src/gapi_lwt.mli

```diff
@@ -0,0 +1,3 @@
+val load : client:string -> callback:(Ojs.t -> unit) -> Ojs.t Lwt.t
```

Comment:
Maybe we want to type this function this way:

```ocaml
val load : client:string -> unit Lwt.t
```

What do you think?

Reply:
The problem is that it will be then impossible to give a "callback" argument in the "load" function, I don't know if this argument is really necessary but it can turn this function less useful for an other application.

Reply:
In addition, if I use "unit Lwt.t" as returned value then I will have to use "unit Lwt.t" as return value of the load function of "base.mli": if I do that I think that then nobody will be able to use the returned value as a numerical or string parameter for the "callback", it could be a problem if someone wants to use the returned value in the callback and not just give a unit.

Reply:
According to the [documentation](https://github.com/google/google-api-javascript-client/blob/master/docs/reference.md#----gapiloadlibraries-callbackorconfig------), it seems that the function `load` does not return any *interesting* value. Anyway, if it was the case, a user of this library would be able to add a *custom callback* after the loading by calling `Lwt.bind` on the returned promise.